frontend/public/components/monitoring/alerting.tsx

```diff
@@ -99,47 +99,62 @@ const ruleURL = (rule: Rule) => `${RuleResource.plural}/${_.get(rule, 'id')}`;
 const pollers = {};
 const pollerTimeouts = {};
 
-const silenceAlert = (alert: Alert) => ({
+const silenceAlert = (alert: Alert, t) => ({
```

Comment:
I would just use the global `i18next.t` instead instead of passing `t` as a parameter. Not all `t`s are the same since they can be associated with a specific namespace.

Reply:
Thanks @spadgett. I changed this and similar functions in this file to use `i18next.t`.